docs/nnx/nnx_basics.md

```diff
@@ -225,22 +225,24 @@ input (scan over layers).
 
```

Comment:
I think we'd want to explain a bit about scan here. What about changing this line to:

> Next lets take a look at a different example, which uses `nnx.vmap` to create a stack of multiple `MLP` layers and `nnx.scan` to iteratively apply each layer of the stack to the input.

Reply:
Thanks!